sample_project/Assets/SampleViewer/Samples/FeatureLayer/Scripts/FeatureLayerInputManager.cs

```diff
@@ -0,0 +1,200 @@
+// Copyright 2024 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class FeatureLayerInputManager : MonoBehaviour
+{
+    public FeatureData FeatureData;
+
+    [SerializeField] private Transform container;
+    private InputActions inputActions;
+    private bool isLeftShiftPressed;
+    private List<GameObject> items = new List<GameObject>();
+    [SerializeField] private Material outlineMat;
+    [SerializeField] private GameObject properties;
+    [SerializeField] private GameObject propertiesView;
+    private TouchControls touchControls;
+    private FeatureLayerUIManager uiManager;
+
+    private IEnumerator AddItemsToScrollView()
+    {
+        foreach (var item in items)
```

Comment:
Quickly selecting between different features can cause an InvalidOperationException for concurrent modification ofthe array.

Reply:
Must not have pushed this but I had this fixed already. Thankfully it's a one-line change.